activities/Calligra.activity/index.html

```diff
@@ -24,6 +24,7 @@
 <script>if (window.module) module = window.module;</script>
 <script src="lib/require.js"></script>
 <script type="text/javascript" src="../../cordova.js"></script>
+<script src="https://cdn.jsdelivr.net/npm/canvas-confetti@1.3.1/dist/confetti.browser.min.js"></script>
```

Comment:
Please copy the confetti file locally so the activity will be able to work offline.

Reply:
While adding confetti file with npm I am getting message 2 low severity vulnerabilties found

Reply:
Include the same file than the one in Tangram activity, there is no message in Tangram activity